SQRLDotNetClientUI/ViewModels/MainMenuViewModel.cs

```diff
@@ -465,5 +468,77 @@ private string GetInstallerByPlatform()
 
             return "";
         }
+
+        /// <summary>
+        /// Allows a user to pick and load or import a sqrl.db file to be used to store application identities and settings
+        /// </summary>
+        private async void ImportDB()
+        {
+            Log.Information("User chose to Import New DB File");
+            var ofdDB = new OpenFileDialog();
+            ofdDB.Title = "Select a valid sqrl.db file";
```

Comment:
Missing localization.

Reply:
Good catch! forgot you put localization in there too!

Reply:
Fixed, ps throw me a bone with the German when you get a chance. I'm not even attempting that long sentence on google translate LoL

Reply:
Sure, will fix it next time I rummage around in the localization file! 😀👍🏻